aerogear/mobile-security-service
# feat(ui): Add top-level app information to app detailed view (AEROGEAR-8850)

## Motivation

https://issues.jboss.org/browse/AEROGEAR-8850

## What

Create the top level app information as shown on the flat. Also did some additional work to use the [Patternfly 4 Grid Layout](http://patternfly-react.surge.sh/patternfly-4/layouts/grid/). Also added this to the landing page.

## Why

To show app information to the user.

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Go to the app detailed view. For example, http://localhost:3000/apps/0890506c-3dd1-43ad-8a09-21a4111a65a6
2. Does it show the top-level app information above the **Deployed Versions** table?

## Checklist:


## Progress
